eval/deployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: ilab
+  name: sdg
+spec:
+  selector:
+    matchLabels:
+      app: sdg
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: sdg
+    spec:
+      containers:
+        - name: container
+          image: quay.io/michaelclifford/test-sdg:0.0.0
+          resources:
+          command:
+            - /bin/bash
+            - '-c'
+            - '--'
+          args:
+            - while true; do sleep 300; done
```

Comment:
should we add `python3.11 eval/run.py` here

Reply:
yeah, sure.